tests/integration/selinux/test_podman_compose_selinux.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-License-Identifier: GPL-2.0
+
+import json
+import os
+import unittest
+
+from tests.integration.test_utils import RunSubprocessMixin
+from tests.integration.test_utils import podman_compose_path
+from tests.integration.test_utils import test_path
+
+
+class TestPodmanCompose(unittest.TestCase, RunSubprocessMixin):
+    def setUp(self):
+        # there must be a source file in the host for volumes type: bind
+        head, tail = os.path.split(podman_compose_path())
+        self.path_to_host_test_file = head + "/host_test_text.txt"
+        self.run_subprocess(["touch", self.path_to_host_test_file])
```

Comment:
probably better simply to create an empty file and add it to git instead of creating untracked files in the repository.